fslib/ui/ui_filepicker.ui

```diff
@@ -47,80 +161,137 @@
+           </property>
+           <property name="text">
+            <string>Search</string>
+           </property>
+          </widget>
+         </item>
          <item>
-          <widget class="QPushButton" name="ui_other_fs">
+          <widget class="QLineEdit" name="filelist_search">
            <property name="sizePolicy">
-            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
+            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
-             <width>32</width>
+             <width>241</width>
+             <height>0</height>
+            </size>
+           </property>
+          </widget>
+         </item>
+         <item>
+          <widget class="QPushButton" name="ui_Action">
+           <property name="enabled">
+            <bool>false</bool>
+           </property>
+           <property name="minimumSize">
+            <size>
+             <width>100</width>
              <height>32</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
-           <property name="text">
-            <string>OTHER</string>
+           <property name="styleSheet">
+            <string notr="true">QPushButton:enabled {
+    background-color: lightgreen;
+    color: black;
+    border-style: outset;
+    border-width: 1px;
+    border-radius: 10px;
+    font: bold 14px;
+    border-color: black;
+    padding: 6px;
+    max-width: 90px;
+}
+
+QPushButton:disabled {
+    background-color: grey;
+    color: black;
+    border-style: outset;
+    border-width: 1px;
+    border-radius: 10px;
+    font: bold 14px;
+    border-color: black;
+    padding: 6px;
+    max-width: 90px;
+}</string>
            </property>
-          </widget>
-         </item>
-         <item>
-          <widget class="QLabel" name="ui_label_Lookin">
            <property name="text">
-            <string>Look in:</string>
+            <string>Open</string>
            </property>
           </widget>
          </item>
          <item>
-          <widget class="QComboBox" name="ui_DirList">
-           <property name="sizePolicy">
-            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
-             <horstretch>0</horstretch>
-             <verstretch>0</verstretch>
-            </sizepolicy>
-           </property>
+          <widget class="QPushButton" name="ui_Cancel">
            <property name="minimumSize">
             <size>
-             <width>380</width>
-             <height>0</height>
+             <width>100</width>
+             <height>32</height>
+            </size>
+           </property>
+           <property name="maximumSize">
+            <size>
+             <width>92</width>
+             <height>16777215</height>
```

Comment:
This seems like a rather large value. These are px, right?

Reply:
Thank you for pointing You are correct height value of 16,777,215 pixels seems unusually large for a QPushButton widget. It appears to be a mistake or an unintended value. Will normalize it.